test/cmd/certs.sh

```diff
@@ -161,6 +197,9 @@ os::cmd::expect_success_and_text \
                                 --expire-days=$((365*3))" \
     'WARNING: .* is greater than 2 years'
 
+os::cmd::expect_success_and_text "cat '${CERT_DIR}/ca.serial.txt'" '08'
```

Comment:
I'd test only first 1-2 iterations. What is the value of testing the same for 8 times?

Reply:
Because different commands are interleaved throughout that are all incrementing this number.